testsuite/MDAnalysisTests/core/test_universe.py

```diff
@@ -161,12 +161,10 @@ def test_Universe_invalidpermissionfile_IE_msg(self, tmpdir):
                                 shell=True)
             else:
                 os.chmod(temp_file, 0o200)
-            try:
+
+            # Issue #3221 match by PermissionError and error number instead
+            with pytest.raises(PermissionError, match="13"):
```

Comment:
Is there a way that we can test for the error code itself as a number instead of checking that the string is included?

Reply:
You could use
```python
import errno

with pytest.raises(PermissionError, match=str(errno.EACCES)):
```

because I get
```python
>>> errno.errorcode[13]
'EACCES'
```

Reply:
Sounds good, I've added `Errno` to the match too so we know we're not just capturing a random number. (I'm assuming the `Errno 13` match is language agnostic, which seems to be the case in #3221, but I don't have a non-English system to test against at the moment).